lib/kitchen/provisioner/policyfile_zero.rb

```diff
@@ -38,15 +38,19 @@ class PolicyfileZero < ChefBase
       # * `deployment_group`: `POLICY_NAME-local`
       # Since it makes no sense to modify these, they are hardcoded elsewhere.
       default_config :client_rb, {}
-      default_config :ruby_bindir, "/opt/chef/embedded/bin"
-
-      # Policyfile mode does not support the `-j dna.json` option to
-      # `chef-client`.
-      default_config :json_attributes, false
+      default_config :json_attributes, true
+      default_config :chef_zero_host, nil
       default_config :chef_zero_port, 8889
 
       default_config :chef_client_path do |provisioner|
-        File.join(provisioner[:chef_omnibus_root], %w[bin chef-client])
+        provisioner.
+          remote_path_join(%W[#{provisioner[:chef_omnibus_root]} bin chef-client]).
+          tap { |path| path.concat(".bat") if provisioner.windows_os? }
```

Comment:
Seems like this is something TK should give us for "free" (though it's fine to merge the PR and consider this note something that could be improved later if it's not).